sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+For me, then, the most essential characteristic of openness is not so much the availability of source code (although this is a prerequisite for other forms of openness), nor the enactment of a specific set of technical governance and legal policies, but the way that collaborative practices performed in the context of an inclusive community can engender artifacts that in some way materially embody these practices. Software that forms, to use McKenzie Wark's term, new creative "vectors for a collective becoming" (_Hacker Manifesto_ 159-60).
+
+## Material Artifacts
+
+As the term suggests, "open source" software development strategies, both socially and technically, emphasize source code as the locus of action and power within computation (Chun 21). Yet even the term itself--open source--is highly contested, particularly by those who identify instead with the "free software" movement. The distinction is marked by differences of legal position and political philosophy. "Free" requires, as an explicit feature of its license, that all modifications or derivations of the source code, including its combination with other works, must be distributed under the same terms as the original (Stallman 32). This is "viral" licensing, where legal requirements spread to everything a piece of software touches. Sharing is legally mandated, never a spontaneous gift. Free software advocates promote this as a legal right to freedom--"free as in speech," the ensconcement of a programmer's right to express themselves via source code, rather than the "free as in beer" of no-cost distribution (Stallman 43).
+
+While I admire aspects of Stallman's project, I have nonetheless tried to intentionally avoid the use of the term "free" here in favour of "open." Partly, this is to avoid what I see as the increasingly blunt ideological rhetoric and overt legalism of many "copyleft" free software advocates who gravitate towards software's technical and logical construction (again, its source code), rather than its social and creative consequences--its _use in practice_. I prefer here to use the term "open community" instead of either "open source" or "free software." This term aims at the possibility that "openness"--to creative expression and personal adaptation of software--might increasingly be conceived of as a characteristic of software artifacts themselves and the communities that form around and within them, not just of source code. In other words, software has the potential, if approached differently, to form a kind of _material_ that can be worked on by means of itself, where the power to create and modify is available to all creators, not only computer programmers (Basman, _If What We Made Were Real_).
```

Comment:
This is astute: `the increasingly blunt ideological rhetoric and overt legalism of many "copyleft" free software advocates who gravitate towards software's technical and logical construction (again, its source code), rather than its social and creative consequences--its _use in practice_.` - I think it properly pinpoints the way the strong technical determinism (for lack of a better way to put it) of Stallman and others can limit a community's capacity to grow and develop - in believing that technical and legal factors are so paramount to the eventual form of community, creation and society (rather than aspects of a more complex web), the FSF has become self-limiting and insular. One can observe a similar narrowness of vision in artistic communities around the unacceptability of certain forms of expression, techniques, etc etc.